larray/core.py

```diff
@@ -3699,11 +3711,9 @@ def ptp(array, *args, **kwargs):
     array : LArray
         Input data.
     axis : None or int or str or Axis, optional
-        Axis along which to find the peaks.
-        The default (`axis` = `None`) is to find the peaks
-        over all axes of the input array (as if the array is flatten).
-        `axis` may be negative, in which case it counts from the last
-        to the first axis.
+        Axis along which to find the peaks. The default (`axis` = `None`) is to find the peaks over all axes of the
+        input array (as if the array was flattened).
+        `axis` may be negative, in which case it counts from the last to the first axis.
```

Comment:
docstring of aggregate function are gonna be generated using a special function and moved in LArray class in a next PR. But OK for now.